backend/blueprint/api/__init__.py

```diff
@@ -19,6 +25,9 @@ def create_app():
     app.config['SECRET_KEY'] = config.POSTGRES['secret']
     app.config['DEBUG'] = config.DEBUG
     app.config.from_object('api.config.mailConfig.MailConfig')
+    app.config.from_object(os.getenv('APP_CONFIG', default='api.config.config.ProductionConfig'))
     CORS(app, resources={r"*": {"origins": "*"}})
+    db.init_app(app)
+    Migrate(app, db)
```

Comment:
What is the use case of shifting it here?? :confused:

Reply:
Because it should be defined here only

Reply:
Moreover, `open-event-server` follows the same approach https://github.com/fossasia/open-event-server/blob/a80b40449ccbfb183d6fb2680fdb36cb53c4ba05/app/__init__.py#L72

Reply:
Them you probably also have seen the blueprint registration, also add the same. Cheers

Reply:
@yashLadha , Sure I will update, thanks

Reply:
Done @yashLadha 👍